docs/variants/msi_z690/cpu-hcl.md

```diff
@@ -2,41 +2,158 @@
 
 ## Contributing
 
-Feel free to test different processors and report your results via
-[email](mailto:contact@dasharo.com) or submit a Pull Request to
-[Dasharo documentation repository](https://github.com/Dasharo/docs) or by using
-[Dasharo issues repository](https://github.com/Dasharo/dasharo-issues/issues).
+* Use [Dasharo Tools Suite HCL
+  report](../../../dasharo-tools-suite/documentation/#hcl-report) to upload
+  report automatically.
+* Create new issue in [Dasharo issues
+  repository](https://github.com/Dasharo/dasharo-issues/issues).
+* Create PR directly to [Dasharo documentation
+  repository](https://github.com/Dasharo/docs).
+* [email](mailto:contact@dasharo.com?subject=CPU HCL report for msi_z690) your
+  `/proc/cpuinfo` or relevant information using following template:
+
+  ```text
+  CPU model:
```

Comment:
How about contributing  to DDR and GPU lists?

Reply:
@macpijan added in https://github.com/Dasharo/docs/pull/463/commits/87c3fcc07b81eb29f05171ee8392a9fbcd9dbc33